baselines/dom.generated.d.ts

```diff
@@ -10439,18 +10439,20 @@ declare var StyleSheetPageList: {
 }
 
 interface SubtleCrypto {
-    decrypt(algorithm: string | Algorithm, key: CryptoKey, data: ArrayBufferView): any;
-    deriveBits(algorithm: string | Algorithm, baseKey: CryptoKey, length: number): any;
-    deriveKey(algorithm: string | Algorithm, baseKey: CryptoKey, derivedKeyType: string | Algorithm, extractable: boolean, keyUsages: string[]): any;
-    digest(algorithm: string | Algorithm, data: ArrayBufferView): any;
-    encrypt(algorithm: string | Algorithm, key: CryptoKey, data: ArrayBufferView): any;
-    exportKey(format: string, key: CryptoKey): any;
-    generateKey(algorithm: string | Algorithm, extractable: boolean, keyUsages: string[]): any;
-    importKey(format: string, keyData: ArrayBufferView, algorithm: string | Algorithm, extractable: boolean, keyUsages: string[]): any;
-    sign(algorithm: string | Algorithm, key: CryptoKey, data: ArrayBufferView): any;
-    unwrapKey(format: string, wrappedKey: ArrayBufferView, unwrappingKey: CryptoKey, unwrapAlgorithm: string | Algorithm, unwrappedKeyAlgorithm: string | Algorithm, extractable: boolean, keyUsages: string[]): any;
-    verify(algorithm: string | Algorithm, key: CryptoKey, signature: ArrayBufferView, data: ArrayBufferView): any;
-    wrapKey(format: string, key: CryptoKey, wrappingKey: CryptoKey, wrapAlgorithm: string | Algorithm): any;
+    decrypt(algorithm: string | RsaOaepParams | AesCtrParams | AesCbcParams | AesCmacParams | AesGcmParams | AesCfbParams, key: CryptoKey, data: ArrayBuffer | ArrayBufferView): PromiseLike<ArrayBuffer>;
```

Comment:
the algorithm parameter is of type `AlgorithmIdentifier`,  which can be defined as `string|Algorithm`. not sure i see the point of enumerating all possible Algorithm types here.